modules/tensor_mechanics/tests/plane_strain/planestrain_test.i

```diff
@@ -0,0 +1,201 @@
+  [./saved_x]
+  [../]
+  [./saved_y]
+  [../]
+
+  [./stress_xx]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+  [./stress_xy]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+  [./stress_yy]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+  [./stress_zz]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+
+  [./strain_xx]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+  [./strain_xy]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+  [./strain_yy]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+  [./aux_strain_zz]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+[]
+
+#[Postprocessors]
+#  [./react_z]
+#    type = MaterialTensorIntegral
+#    tensor = stress
+#    index = 2
+#    use_displaced_mesh = true
+#  [../]
+#[]
+
+[Kernels]
+  [./TensorMechanics]
+    disp_x = disp_x
+    disp_y = disp_y
+  [../]
+[]
+
+[AuxKernels]
+  [./stress_xx]
+    type = RankTwoAux
+    rank_two_tensor = stress
+    variable = stress_xx
+    index_j = 0
+    index_i = 0
+  [../]
+  [./stress_xy]
+    type = RankTwoAux
+    rank_two_tensor = stress
+    variable = stress_xy
+    index_j = 1
+    index_i = 0
+  [../]
+  [./stress_yy]
+    type = RankTwoAux
+    rank_two_tensor = stress
+    variable = stress_yy
+    index_j = 1
+    index_i = 1
+  [../]
+  [./stress_zz]
+    type = RankTwoAux
+    rank_two_tensor = stress
+    variable = stress_zz
+    index_j = 2
+    index_i = 2
+  [../]
+
+  [./strain_xx]
+    type = RankTwoAux
+    rank_two_tensor = total_strain
+    variable = strain_xx
+    index_j = 0
+    index_i = 0
+  [../]
+  [./strain_xy]
+    type = RankTwoAux
+    rank_two_tensor = total_strain
+    variable = strain_xy
+    index_j = 1
+    index_i = 0
+  [../]
+  [./strain_yy]
+    type = RankTwoAux
+    rank_two_tensor = total_strain
+    variable = strain_yy
+    index_j = 1
+    index_i = 1
+  [../]
+  [./strain_zz]
+    type = RankTwoAux
+    rank_two_tensor = total_strain
+    variable = aux_strain_zz
+    index_j = 2
+    index_i = 2
+  [../]
+[]
+
+[Functions]
+  [./pull]
+    type = PiecewiseLinear
+    x='0     1  100'
+    y='0  0.00 0.00'
+  [../]
+[]
+
+[BCs]
+  [./bottomx]
+    type = PresetBC
+    boundary = 1
+    variable = disp_x
+    value = 0.0
+  [../]
+  [./bottomy]
+    type = PresetBC
+    boundary = 1
+    variable = disp_y
+    value = 0.0
+  [../]
+[]
+
+[Materials]
+  [./small_strain]
+    type = ComputePlaneSmallStrain
+    block = 1
+    disp_y = disp_y
+    disp_x = disp_x
+  [../]
+  [./linear_stress]
+    type = ComputeLinearElasticStress
+    block = 1
+  [../]
+  [./elasticity_tensor]
+    type = ComputeIsotropicElasticityTensor
+    block = 1
+    poissons_ratio = 0.3
+    youngs_modulus = 1e6
+  [../]
+[]
+
+[Executioner]
+  type = Transient
+
+  solve_type = PJFNK
+  line_search = none
+
+
+# controls for linear iterations
+  l_max_its = 100
+  l_tol = 1e-4
+
+# controls for nonlinear iterations
+  nl_max_its = 15
+  nl_rel_tol = 1e-10
+  nl_abs_tol = 1e-5
```

Comment:
Does this test have a hard time converging below 1e-5? If not then this line should be removed otherwise this test will likely diff on other platforms. The default Exodiff tolerances are in around 5e-6.